### PR TITLE
kata-deploy: Fix tdx_not_supported call

### PR DESCRIPTION
**Pull request by ldoktor**
the `tdx_not_supported_warning` function does not exists, the `tdx_not_supported` should be called instead.

Fixes: #9628

### CODE REVIEW

File: tools/packaging/kata-deploy/kata-deploy/base/kata-deploy.yaml [hunk 1]
```diff
@@ -17,7 +17,7 @@ spec:
       hostPID: true
       containers:
         - name: kube-kata
-          image: quay.io/kata-containers/kata-deploy:latest
+          image: quay.io/kata-containers/kata-deploy-ci:kata-containers-latest
```
Comment (gkurz) [inline tools/packaging/kata-deploy/kata-deploy/base/kata-deploy.yaml:20]: This change doesn't seem to be related to the PR or I'm missing something ?

File: tools/packaging/kata-deploy/scripts/kata-deploy.sh [hunk 2]
```diff
@@ -273,7 +273,7 @@ function install_artifacts() {
 					esac
 					;;
 				*)
-					tdx_not_supported_warning
+					tdx_not_supported ${ID} ${VERSION_ID}
```
Comment (gkurz) [inline tools/packaging/kata-deploy/scripts/kata-deploy.sh:276]: This was introduced by commit a9720495deb2 . @fidencio can you confirm ?

Comment (fidencio) [reply tools/packaging/kata-deploy/scripts/kata-deploy.sh:276]: Yep, my bad :-/

File: tools/packaging/kata-deploy/kata-deploy/base/kata-deploy.yaml (hunk 1 above)
Comment (fidencio) [reply tools/packaging/kata-deploy/kata-deploy/base/kata-deploy.yaml:20]: I second this one!

Comment (ldoktor) [reply tools/packaging/kata-deploy/kata-deploy/base/kata-deploy.yaml]: (good old `git commit -as` without reviewing the content...)

**Review by fidencio — APPROVED** [review]
lgtm, thanks @ldoktor!

**Review by gkurz — APPROVED** [review]
LGTM. Thanks @ldoktor !
